contrib/forms/assessment_intake/view.php

```diff
@@ -376,4 +394,4 @@
 </form>
 <?php
 formFooter();
-?>
+?> 
```

Comment:
good point, however need to ensure that then drop the closing php tag (or else it breaks things, such as the html2pdf conversion when creating patient reports). Several times have had to search code to remove the spacing after the php closing tags:
**2.2. Files**
All PHP files MUST use the Unix LF (linefeed) line ending.
All PHP files MUST end with a single blank line.
**The closing ?> tag MUST be omitted from files containing only PHP.**